src/raylib.h

```diff
@@ -194,6 +194,10 @@
 #define MAGENTA    CLITERAL(Color){ 255, 0, 255, 255 }     // Magenta
 #define RAYWHITE   CLITERAL(Color){ 245, 245, 245, 255 }   // My own White (raylib logo)
 
+// Filter string used in ScanDirectoryFiles, ScanDirectoryFilesRecursively and
+// LoadDirectoryFilesEx to include directories in the result array
+#define FILTER_FOLDER "/DIR"
```

Comment:
I prefer to avoid exposing this define, it can be defined internally in `rcore` and just document as an option in the function

Reply:
I can make the change, no problem.
Just to make sure: People would call it like this (with the "/DIR" option mentioned in documentation)?
LoadDirectoryFilesEx("somepath", "/DIR;.c", false);

The only minor issue I see with this, is that documentation has to be updated separately from the code then. If you ever want to change the "/DIR" string to something else, you will have to make sure to edit the documentation accordingly. This would also break existing code, while using an exposed define would always "just work".
Personally I would rather expose the define, but it's your library so you make the call.

If you are okay with that, I will make the requested change.

Reply:
We could keep the define but internal to `rcore` module and configurable **only at compile time** through `config.h`, like many other options in raylib.

Reply:
Okay I made the requested changes and pushed them to the branch.